webknossos/webknossos/skeleton/group.py

```diff
@@ -61,64 +79,87 @@ def add_graph(
 
     @property
     def children(self) -> Iterator[GroupOrGraph]:
+        """Returns all (immediate) children (groups and graphs) as an iterator."""
         return (child for child in self._children)
 
+    @property
+    def graphs(self) -> Iterator[GroupOrGraph]:
+        """Returns all (immediate) graph children as an iterator.
+        Use flattened_graphs if you need also need graphs within subgroups."""
+        return (child for child in self._children if isinstance(child, Graph))
+
+    @property
+    def groups(self) -> Iterator[GroupOrGraph]:
+        """Returns all (immediate) group children as an iterator.
+        Use flattened_groups if you need also need groups within subgroups."""
+        return (child for child in self._children if isinstance(child, Group))
```

Comment:
I think those properties are great additions. However, I'd rather store the graphs and groups separate from each other and simply combine them in `children`, which saves all the `isinstance` checks.

Reply:
good point :+1: simplifies the code, too.